### ISSUE TITLE
Cannot read property 'remote' of undefined

### ISSUE THREAD

**Issue by turt2live** [Half-Shot/matrix-appservice-discord#399]
```
Feb-16 07:11:56.158 [DiscordAS] verbose: matrix-appservice-bridge [fv24zg5oe6g00] FAILED (516ms) TypeError: Cannot read property 'remote' of undefined
    at MatrixRoomHandler.<anonymous> (/src/matrixroomhandler.ts:184:38)
    at Generator.next (<anonymous>)
    at /build/src/matrixroomhandler.js:22:71
    at new Promise (<anonymous>)
    at __awaiter (/build/src/matrixroomhandler.js:18:12)
    at MatrixRoomHandler.OnEvent (/build/src/matrixroomhandler.js:121:16)
    at Object.<anonymous> (/src/discordas.ts:111:74)
    at Generator.next (<anonymous>)
    at fulfilled (/build/src/discordas.js:19:58)
    at processTicksAndRejections (internal/process/next_tick.js:81:5)
```

**Comment by Half-Shot** [comment]
:(

**Comment by Sorunome** [comment]
Is this when bridging? if so, it is likely the same thing as #436

**Comment by turt2live** [comment]
It happens much more frequently than that - several times a minute. I hope people aren't bridging that often ._.

**Comment by Sorunome** [comment]
it seems to be the context stuff, which would be fixed in #438

**Comment by Sorunome** [comment]
fixed
